recipes/festvox-kallpc16k/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "festvox-kallpc16k" %}
+{% set version = "2.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://festvox.org/packed/festival/2.5/voices/festvox_kallpc16k.tar.gz
+  sha256: 809c4ab5ed9e4df4a658b58d5c56fe35055723f00d81a238168f5a1ebdaed08c
+
+build:
+  skip: True  # [not linux]
+  number: 0
```

Comment:
Why is festvox-kallpc16k platform specifc instead of generic?

Reply:
@carterbox I tried making it generic, but in that case it depends on festival on all platforms, however only linux64 has the package. So what is the correct way of specifying this? It doesn't make sense to me to add the festival dependency only for linux and make the other platforms without any.

Reply:
The reason why I made this package depend on the festival binary is because this kind of dependency is present in both Ubuntu and Arch Linux.

Reply:
> however only linux64 has the package.

That's OK. Noarch is a description of the contents of the package, not of its dependencies.

Reply:
This package will just be uninstallable by win/osx until the festival package is build for those platforms. That's fine. If it's not fine, then this package shouldn't depend on festival. i.e. You can remove those depenencies if someday someone wants to use these files with another program.